README.md

```diff
@@ -36,12 +36,16 @@ Contains Protocol Buffer files describing messages sent to/received from the ARI
 - **reorder-frame** - sample program/unit test that illustrates reordering an ARIS frame; this makes use of the reordering code in aris-integration-sdk/common/code/.
 - **vc-using-framestream** - sample program that implements a full range of commands, records frames to a file, and, optionally, uses multicast for frame acquisition.
 
-### Tools
+### ARIS Tools
 
 **[aris-integration-sdk/tools](tools/)**: contains the following:
 
 - **arislog** - this Windows console application displays messages received from syslog relays, which may be useful for debugging client code. Note that a successful connection must be made after power on to set up forwarding of ARIS syslog messages to the client PC. Note also that this program will display **all** syslog messages forwarded to the PC, even from other applications.
 
+### External Tools
+
+This repository now makes use of **vcpkg** for building **arislog** and **vc-using-framestream**. **vcpkg** is an external tool that cannot be pinned to a specific version when fetching a library, so there is a slight chance you may find an issue when building these files. That said, our daily build should catch any issue pretty quickly.
```

Comment:
It may be still in your best interest to not pin the vcpkg version, but it is possible to do so if you'd like. Each commit in Vcpkg corresponds to a SHA512-hash-checked set of libraries for exactly this reason, so if you specify the commit of Vcpkg to use, you effectively specify the _exact_ versions of every library.

Some possible ways to do this:
1. Just tell users the latest hash you've tested against. They can checkout that vcpkg version if they hit issues with latest.
2. Fork vcpkg on GitHub and just pull from the upstream repo whenever you want to test+update. Then, you could tell users to grab a branch on your fork for the stable version.
3. Add vcpkg as a submodule (could be combined with 2). Users could either supply an external vcpkg or use `git submodule init` + `git submodule update` to pull the exact vcpkg version that this commit was tested against.
4. If you want to use prebuilt binaries, you could look at our `export` command [1], which can build a prebuilt archive of all your transitive dependencies.

[1] https://vcpkg.readthedocs.io/en/latest/specifications/export-command/